build.sbt

```diff
@@ -443,11 +451,25 @@ lazy val scaldingParquetScrooge = module("parquet-scrooge")
         exclude("com.twitter.elephantbird", "elephant-bird-core"),
        "com.twitter" %% "scrooge-serializer" % scroogeVersion,
       "org.apache.hadoop" % "hadoop-client" % hadoopVersion % "provided",
-      "com.novocode" % "junit-interface" % "0.11" % "test",
       "junit" % "junit" % junitVersion % "test"
+    )
+).dependsOn(scaldingParquetCascading % "compile->compile;test->test", scaldingParquetScroogeFixtures % "test->test")
 
+lazy val scaldingParquetScrooge = module("parquet-scrooge")
```

Comment:
I separated these so that the cascading schemes go in a different sub-project. parquet-scrooge, after this change would contain only the ParquetScrooge trait and related time pathed sources (which need scalding-core). If this is too thin to be its own sub-project, then I can merge back the two.